architecture/2025-04-compute-platform-extensibility.md

```diff
@@ -0,0 +1,505 @@
+
+This document describes the initial integration with ACI, which is built upon an earlier demo capability. Our longer-term approach will provide an extensibility capability to Radius for deploying to other compute platforms, which will be described in a separate document.
+
+## Terms and definitions
+
+* ACI: Azure Container Instances, a service that allows users to run Docker containers on Azure without managing servers.
+
+## Objectives
+
+* To integrate Azure Container Instances (ACI) with Radius, enabling users to deploy and manage their applications and containers in ACI environments.
+* To provide a seamless user experience for deploying Radius applications to ACI, including environment setup, application definition, and deployment.
+* To ensure that the initial features are fully functional and ready for demonstration.
+
+> **Issue Reference:** <!-- (If appropriate) Reference an existing issue that describes the feature or bug. -->
+
+### Goals
+
+* Implement the capability for the Radius control plane hosted in Kubernetes to deploy to ACI.
+* Enable users to set up a Radius Environment targeted at ACI using environment definition files (e.g., env.bicep).
+* Allow users to deploy Radius applications to ACI using application definition files (e.g., app.bicep).
+* Support the deployment of application containers to ACI, ensuring that container definitions are platform-agnostic.
+* Allow users to apply ACI-specific configurations to containers using extension properties.
+* Adding and configuring Dapr sidecars to ACI containers.
+* A default implementation of Azure Gateways on ACI.
+* Defining and creating Secret Stores for use in ACI containers. (The implementation is TBD: KeyVault or Dapr.)
+
+### Non goals
+
+* Hosting the Radius control plane in ACI. The control plane will continue to be hosted in Kubernetes.
+* Defining and creating custom ingress for use in ACI containers.
+* Defining Extenders/UDT for use in ACI containers.
+* Providing a managed Radius offering in ACI.
+
+### User scenarios (optional)
+
+#### User story 1
+
+As a user, I can use the instance of the Radius control plane hosted in my Kubernetes cluster to set up an ACI Environment and target that ACI environment to deploy my Applications and Containers.
+
+#### User story 2
+
+As a user, I can define a new Radius Environment that specifies ACI as the underlying compute platform by creating a new Environment definition file (e.g., env.bicep) and specifying the necessary settings before deploying the environment using rad deploy env.bicep. Applications and container resources that I deploy into this Environment will be targeted at my ACI cluster.
```

Comment:
I see now that you are using applications and containers as in Applications.Core/applications and Applications.Core/containers.